### PR TITLE
Fixes upbound/upjet-provider-template#37

### PR DESCRIPTION
**Pull request by etesami**
This PR fixes the #37 issue.

### CODE REVIEW

**Comment by etesami** [comment]
@turkenf can you take a look?

File: hack/prepare.sh [hunk 1]
```diff
@@ -11,13 +11,11 @@ git grep -l 'template' -- ${REPLACE_FILES} | xargs sed -i.bak "s/upjet-provider-
 # shellcheck disable=SC2086
 git grep -l 'template' -- ${REPLACE_FILES} | xargs sed -i.bak "s/template/${PROVIDER_NAME_LOWER}/g"
 # shellcheck disable=SC2086
-git grep -l "upbound/provider-${PROVIDER_NAME_LOWER}" -- ${REPLACE_FILES} | xargs sed -i.bak "s|upbound/provider-${PROVIDER_NAME_LOWER}|${ORGANIZATION_NAME}/provider-${PROVIDER_NAME_LOWER}|g"
-# shellcheck disable=SC2086
```
Comment (turkenf) [inline hack/prepare.sh:15]: Let's not remove these lines so that files other than Makefile are not affected.

File: hack/prepare.sh [hunk 2]
```diff
@@ -11,13 +11,11 @@ git grep -l 'template' -- ${REPLACE_FILES} | xargs sed -i.bak "s/upjet-provider-
...
 git grep -l 'Template' -- ${REPLACE_FILES} | xargs sed -i.bak "s/Template/${PROVIDER_NAME_NORMAL}/g"
 # We need to be careful while replacing "template" keyword in go.mod as it could tamper
 # some imported packages under require section.
 sed -i.bak "s|upbound/upjet-provider-template|${ORGANIZATION_NAME}/provider-${PROVIDER_NAME_LOWER}|g" go.mod
-sed -i.bak "s|PROJECT_REPO := github.com/upbound/|PROJECT_REPO := github.com/${ORGANIZATION_NAME}/|g" Makefile
+sed -i.bak "s|upbound\/\$(PROJECT_NAME)|${ORGANIZATION_NAME}\/provider-${PROVIDER_NAME_LOWER}|g" Makefile
```
Comment (turkenf) [inline hack/prepare.sh:18]:
```suggestion
sed -i.bak "s|PROJECT_REPO ?= github.com/upbound/|PROJECT_REPO :?= github.com/${ORGANIZATION_NAME}/|g" Makefile
```

I think this issue appeared after variables were allowed to be set externally in [this PR](https://github.com/upbound/upjet-provider-template/pull/30/files). So I think just the above change will suffice. What do you think?

File: hack/prepare.sh (hunk 1 above)
Comment (etesami) [reply hack/prepare.sh:15]: Sure @turkenf, I agree.

File: hack/prepare.sh (hunk 2 above)
Comment (etesami) [reply hack/prepare.sh:18]: This would work. However, it seems there is a typo where `:?=` should be `?=`?

Comment (turkenf) [reply hack/prepare.sh:18]: Yes, you are right should be `?=`

**Review by turkenf — APPROVED** [review]

**Pull request merged**